mods/cnc/maps/gdi03/map.yaml

```diff
@@ -823,15 +823,35 @@ Rules:
 			Amount: -10
 		-Capturable:
 	HQ:
+		Buildable:
+			Prerequisites: ~disabled
+	NOHQ:
+		Inherits: HQ
+		Buildable:
+			Prerequisites: proc
+		Tooltip:
+			Description: Provides an overview of the battlefield.\n  Requires power to operate.
+		-AirstrikePower:
+		RenderBuilding:
+			Image: hq
+	airstrike.proxy:
```

Comment:
That's the way it is in the original missions, though: http://youtu.be/MnJJAag-S9c?t=3m6s

Note that you don't have a conyard in gdi05. If you had, you probably would be able to build a radar, too.  However, in the original, the grenadiers don't require the HQ, but that's alright with me.